controller/controller.go

```diff
@@ -0,0 +1,124 @@
+package controller
+
+import (
+	"compress/gzip"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/brave/go-sync/auth"
+	"github.com/brave/go-sync/command"
+	"github.com/brave/go-sync/datastore"
+	"github.com/brave/go-sync/sync_pb"
+	"github.com/brave/go-sync/timestamp"
+	"github.com/go-chi/chi"
+	"github.com/golang/protobuf/proto"
+	"github.com/rs/zerolog/log"
+)
+
+// SyncRouter add routers for command and auth endpoint requests.
+func SyncRouter(datastore datastore.Datastore) chi.Router {
+	r := chi.NewRouter()
+	r.Post("/command/", Command(datastore))
+	r.Post("/auth", Auth(datastore))
+	r.Get("/timestamp", Timestamp)
+	return r
+}
+
+func sendJSONRsp(body []byte, w http.ResponseWriter) {
```

Comment:
Suggest you add the response code here too, so you can do `WriteHeader(<statuscode>)` as you are sending

Reply:
addressed in https://github.com/brave/go-sync/pull/1/commits/7a1ea988c0e5822347baa508ced53b75d4e425f0